modules/tinymce/src/core/test/ts/browser/content/EditorContentTest.ts

```diff
@@ -241,6 +241,13 @@ describe('browser.tinymce.core.content.EditorContentTest', () => {
           assertEventsContentType();
         });
 
+        it('TINY-9140: Insert content without firing events', () => {
```

Comment:
Seems to be more properties that the merge now includes should we add test for more of them. Or add a follow up ticket to do so if we are pressed for time.

Reply:
I created a Jira: TINY-9143